gluster_exporter/main.go

```diff
@@ -32,17 +31,79 @@ func registerMetric(name string, fn func(), intervalSeconds int64) {
 	glusterMetrics = append(glusterMetrics, glusterMetric{name: name, fn: fn, intervalSeconds: time.Duration(intervalSeconds)})
 }
 
-func getPeerID() string {
-	return *peerid
+func getWorkingDir( glusterMgmt *string ) string {
+        if( *glusterMgmt == "glusterd2" ) {
+                return "/var/lib/glusterd2"
+        } else {
+                return "/var/lib/glusterd"
+        }
+}
+
+func getPeerID( gluster_workdir *string ) string {
```

Comment:
If caching the return value is planned, do not add here. Add caching in `parseGd1PeerID` and `parseGd2PeerID` functions